### DISCUSSION TITLE
怎么使用mouselnc手势启动exe软件

### DISCUSSION THREAD

**Discussion by qqqqcat1222** [shuax/MouseInc.Settings#386] [General]
【已解决】大佬好，感谢大佬的软件，想问一下怎么使用mouselnc手势启动exe软件，网上搜索了好多文章都没有找到相关的设置方法，麻烦告知下，谢谢。

**Comment by Seryta** [comment]
![image](https://user-images.githubusercontent.com/31036794/189022669-4f736dc6-1066-422f-b91d-8cf6e6b92200.png)

**Reply by qqqqcat1222** [reply]
可以了，感谢大佬

**Comment by Ctrl-Ling** [comment]
https://docs.shuax.com/MouseInc/#/action#execute
